parse_test.py

```
@@ -0,0 +1,19 @@
+"""
+parse_test.py – Unit tests for complex query parsing.
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)